dev/sample_indexes/indexes-metadata.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
I haven't added any code to verify these SHAs after copying them, but I figured it'd be valuable to at least have the SHAs noted down.